src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Core/Builder/ScaffoldRunnerBuilder.cs

```diff
@@ -72,13 +75,42 @@ private void AddDefaultServices()
 
     private void AddCoreServices()
     {
-        Services.AddLogging();
-        Logging.AddCleanConsoleFormatter();
+        //get some logging properties from environment variables
+        var isVerboseEnabled = Environment.GetEnvironmentVariable(ScaffolderConstants.ENABLE_VERBOSE_LOGGING)?.Equals("true", StringComparison.OrdinalIgnoreCase) == true;
```

Comment:
Since we do this `[string]?.Equals("true", StringComparison.OrdinalIgnoreCase) == true` check at least three times in this PR (maybe other places too?) maybe we should create another extension method like the `GetEnvironmentVariableAsBool` one we have? Looks like maybe none of these are using IEnvironmentService - if they can't be changed, I guess they could be extension methods on strings. Or maybe just a helper function so as not to pollute the string intellisense...

Reply:
yep an extension method would do, i'll try to see if adding IEnvironmentService makes sense first.

Reply:
using a helper, fixed